Tools/API/1.0.0/Unity.InputSystem.api

```diff
@@ -17,7 +17,7 @@ namespace Unity.XR.GoogleVr
         [UnityEngine.InputSystem.Layouts.InputControl] [UnityEngine.Scripting.Preserve] public UnityEngine.InputSystem.Controls.ButtonControl volumeDown { get; }
         [UnityEngine.InputSystem.Layouts.InputControl] [UnityEngine.Scripting.Preserve] public UnityEngine.InputSystem.Controls.ButtonControl volumeUp { get; }
         public DaydreamController() {}
-        protected virtual void FinishSetup();
```

Comment:
Seems the pattern here is to change from `virtual` to `override`.  Is it just the way it is for the new format of  "com.unity.coding": "0.1.0-preview.24" or is there any other reason behind it? Anyway, this file was regenerated to meet the new format, correct?

Reply:
It was .24 change, it's probably just to mark an overload with `override` instead of `virtual`